code/__DEFINES/psi.dm

```diff
@@ -0,0 +1,34 @@
+#define PSI_COERCION           "coercion"
```

Comment:
might be a bit confusing having dspawn psi and normal psi, psy could work beter tbh